package-diff

```diff
@@ -32,81 +37,166 @@ VERSION_B="$2"
 FILESONLY="${FILESONLY-0}"
 CUTKERNEL="${CUTKERNEL-0}"
 CALCSIZE="${CALCSIZE-0}"
+WITHNEWSTUFF="${WITHNEWSTUFF-0}"
```

Comment:
Could we make 1 the default? With the fallback in place it's ok, or?

Reply:
Tested and seems to work, I would use it directly to benefit from the correct filtering (but maybe keep the variable around?).

Reply:
Done. At first I thought about it as an opt-in feature, instead of opt-out. But having fallbacks in place indeed makes it sensible to enable this stuff by default. Note that the current releases don't have those new report files, so our CI will be falling back to the old reports until we do new releases of LTS, stable, beta and alpha.